### DISCUSSION TITLE
FR: Slideouts

### DISCUSSION THREAD

**Discussion by wmdhosting** [wbrowar/craft-3-guide#77] [Ideas and Feature Requests]
Hey :)

It will be great if posiblle that we can load..  Guides in slideouts.. 

https://dev-diary.newism.com.au/posts/creating-craft-cms-slideouts.html

**Comment by wbrowar** [comment]
@wmdhosting I saw Leevi’s post and saw his updates on Twitter and was thinking the same thing 😀

No promises on when I can get to it, but I will plan on this getting added in.

**Comment by wbrowar** [comment]
@wmdhosting I finally got a chance to implement this and it turned out to work just as I had hoped. This will be out later this month in Guide 3.3:

https://twitter.com/wbrowar/status/1568982414373928960?s=21&t=w1SYASAILfbUAMJfUWEB1w

**Comment by wmdhosting** [comment]
OK great  :) thx

**Comment by wbrowar** [comment]
Finally available in Guide 5. Thanks again for your patience as this took a long time to figure out and to see it to completion: https://wbrowar.com/article/code/guide-5
